Project1/TxtFile.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Project1
+{
+    internal class TxtFile
+    {
+        string path = "C:\\Users\\wikto\\Desktop\\file.txt";
+        public void ViewTheFile()
+        {
+            string readText = File.ReadAllText(path);
+            Console.WriteLine(readText);
+        }
+        public void SaveText(string text)
+        {
+            StreamWriter sw = new StreamWriter(path, true);
```

Comment:
To moglibyśmy zrobić w konstruktorze. Tą inicjalizację. Konstuktory traktujmy jako takie metody gdzie będziemy określać obiekty potrzebne do wykonywania naszych działań. Coś ala taka konfiguracja potrzebna do poprawnego działania danej klasy